### ISSUE TITLE
Fixed positioned elements inside the ng-scrollbar component aren't behaving properly

### ISSUE THREAD

**Issue by adrianbenjuya** [MurhafSousli/ngx-scrollbar#24]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ x ] feature request
- [ ] question
```

### OS and Version?

Windows 10, macOS Sierra

### Versions

Angular CLI: 1.7.4
Node: 6.11.2
OS: darwin x64
Angular: 5.2.10
... animations, common, compiler, compiler-cli, core, forms
... http, platform-browser, platform-browser-dynamic, router

@angular/cdk: 5.2.5
@angular/cli: 1.7.4
@angular/language-service: 5.2.0
@angular/material: 5.2.5
@angular-devkit/build-optimizer: 0.3.2
@angular-devkit/core: 0.3.2
@angular-devkit/schematics: 0.3.2
@ngtools/json-schema: 1.2.0
@ngtools/webpack: 1.10.2
@schematics/angular: 0.3.2
@schematics/package-update: 0.3.2
typescript: 2.4.2
webpack: 3.11.0

### Repro steps

Add any fixed-positioned element inside the ng-scrollbar component. It will be positioned relative to the scrolling container and it will scroll with it instead of being fixed to the screen as the position: fixed behavior should be.

### Desired functionality

The fixed-positioned element inside the ng-scrollbar component shouldn't scroll with it and it should be positioned relative to the screen

**Comment by MurhafSousli** [comment]
Can you reproduce it in [stackblitz](https://stackblitz.com/edit/ngx-scrollbar)

**Comment by adrianbenjuya** [comment]
Of course, [here you have](https://stackblitz.com/edit/ngx-scrollbar-3vbdm6)

**Comment by adrianbenjuya** [comment]
Hi! I'm a little late, but here I give you a [fixed example]( https://stackblitz.com/edit/ngx-scrollbar-1nlmhy) of what I'm trying to do. I just added a new css rule to remove `transform: translate3d()` from .ng-scrollbar-view

The `position: sticky` solution doesn't fix it because the red square is still inside the scroll container, and it should be outside it. It is due to the `transform: translate3d()` that creates a new local coordinate system (you can look this [answer](https://stackoverflow.com/a/15256339/2684353) for more information)
